presto-docs/src/main/sphinx/installation/jmx.rst

```diff
@@ -0,0 +1,61 @@
+===================
+Monitoring with JMX
+===================
+
+Presto exposes a large number of different metrics via the Java Management
+Extensions (JMX). JConsole, supplied with the JDK, VisualVM, and many other
+tools can be used to access the metrics in a client application. Many monitoring
+solutions support JMX.
+
+You can configure the ports used by JMX in ``jvm.config`` as described in
+:doc:`/installation/deployment`.
+
+Alternatively you can use the :doc:`/connector/jmx` and query the metrics using
+SQL.
+
+Many of these JMX metrics are a complex metric object such as a ``CounterStat``
+that has a collection of related metrics. For example, ``InputPositions`` has
+``InputPositions.TotalCount``, ``InputPositions.OneMinute.Count``, and so on.
+
+The metrics can be grouped into a number of topics and the following sections
+describe a subset of all available metrics.
+
+JVM
+---
+
+* Heap size: ``java.lang:type=Memory:HeapMemoryUsage.used``
+* Thread count: ``java.lang:type=Threading:ThreadCount``
+
+Presto Cluster and Nodes
+------------------------
+
+* Active nodes ``presto.failureDetector:name=HeartbeatFailureDetector:ActiveCount``
+* Free memory (general pool): ``presto.memory:type=ClusterMemoryPool:name=general:FreeDistributedBytes``
+* Blocked Nodes: ``presto.memory:type=ClusterMemoryPool:name=general:BlockedNodes``
+* Number of leaked queries: ``presto.memory:name=ClusterMemoryManager:NumberOfLeakedQueries``
```

Comment:
that kind of suggests "leaking queries" is something normal and one should monitor this. i believe this is not the case

can we replace this example with something more useful, or remove?

Reply:
I will remove .. I think in the future we should expand this to give some real advice on what to look out for but that goes beyond the scope I can tackle now